docs/source/api/v2/cdns_dnsseckeys_generate.rst

```diff
@@ -32,7 +32,6 @@ Request Structure
 :effectiveDate:         An optional string containing the date and time at which the newly-generated :abbr:`ZSK (Zone-Signing Key)` and :abbr:`KSK (Key-Signing Key)` become effective, in :RFC:`3339` format. Defaults to the current time if not specified.
 :key:                   Name of the CDN
 :kskExpirationDays:     Expiration (in days) for the :abbr:`KSKs (Key-Signing Keys)`
-:name:                  Domain name used by the CDN
```

Comment:
I don't think we can make this breaking change to the v2 API, as it's already been released

Reply:
It's not really a breaking change, it's a completely unused field that has no effect on behavior/data whatsoever. API users can continue to send it, it will just be totally ignored (as it basically was before, except without the unnecessary validation).

Reply:
Oh, this endpoint only has a POST method. I wouldn't have removed it even if it's no longer used, personally, but I suppose it's not a big deal.

Reply:
I know, but I think we should reserve the right to remove things like this when it makes sense.